test/babel-transform-constants.mjs

```diff
@@ -15,31 +15,31 @@ test('babel-transform-constants', function (t) {
   t.test('cjs', function (t) {
     t.equal(
       transform(
-        'var codes = require("../lib/character/codes.js")\nconsole.log(codes.carriageReturn)'
+        'var codes = require("../lib/character/codes.mjs")\nconsole.log(codes.carriageReturn)'
```

Comment:
One q: as this is the `cjs` test and a require, I guess these should remain `.js`?

Otherwise: all looks good.

I’ll leave this open for a couple days so other folks can review.

After that I’ll do a bit of refactoring.

And then, ready for a minor release I guess? Anything else?

Reply:
Just to pass tests. I don't think require logic is necessary here now.

I think all is done.

Reply:
ah right, because there is no `.js` in `lib/`. Unfortunate.
There is one in `dist/` though, but that would mean changing the constants plugin, which can be left for another time